managed_resource/provider/nhncloud.yaml

```diff
@@ -0,0 +1,9 @@
+---
+provider: nhncloud
+name: NHN Cloud
+color: '#FF9900'
+icon: 'https://static.toastoven.net/toast/resources/img/logo_nhn_cloud_color.svg'
```

Comment:
Provider icon is used at Service Account. Did you test in console page?

![image](https://github.com/cloudforet-io/plugin-nhncloud-inven-collector/assets/3993088/beb9fc69-a098-4dbc-9327-d8b7f3d10ea6)

Reply:
@choonho I create provider and there is no redering issue.
But it is hard to identify provider via icon. 
Should I use another icon?

Reply:
<img width="987" alt="image" src="https://github.com/cloudforet-io/plugin-nhncloud-inven-collector/assets/55343124/c7befc7c-1a13-4dfb-9bc4-f43747dfc3c6">

Reply:
@choonho 
<img width="1026" alt="image" src="https://github.com/cloudforet-io/plugin-nhncloud-inven-collector/assets/55343124/b42a3301-f876-428d-b143-9edcc0217418">
How about not logo but simbol?